Add Combo article (#3192)
ppy/osu-wiki: wiki/Beatmapping/Combo/en.md

---
tags:
    - Combo Color
    - Colorhaxing
    - Colourhaxing
    - Color Haxing
    - Colour Haxing

>     - NC

Reviewer: ```suggestion
  - Combo Color
  - Colorhaxing
  - Colourhaxing
  - Color Haxing
  - Colour Haxing
  - NC
```

---

# Combo

> *For the element refered to as 'combo' in gameplay, see: [Score Multiplier](/wiki/Score)*

Reviewer: ```suggestion
*Not to be confused with [Score Multiplier](/wiki/Score).*
```

Combos refer to groupings of [hit objects](/wiki/Hit_Objects) done according to musical representation. Groupings are most commonly done according to verses, measures, vocal phrases and/or instrument scripts. Different combos are separated by **New Combos**, or ***NCs***. New combos are made with the `New Combo` button (or the shortcut `Q`) on the right of the `Design` tab in the [beatmap editor](/wiki/Beatmap_Editor). How combos are displayed in gameplay depends on the currently enabled [gamemode](/wiki/Game_Modes).

> ## Combo Colour

Reviewer: ```suggestion
## Combo colour
```

> *For regulations surrounding combo colours, see [Ranking Criteria](/wiki/Ranking_Criteria)*

Reviewer: ```suggestion
*For regulations surrounding combo colours, see: [Ranking Criteria](/wiki/Ranking_Criteria)*
```
